subsys/bluetooth/host/buf.c

```diff
@@ -41,9 +34,7 @@ LOG_MODULE_REGISTER(bt_buf, CONFIG_BT_LOG_LEVEL);
  * the HCI transport to fill buffers in parallel with `bt_recv`
  * consuming them.
  */
-#define SYNC_EVT_SIZE NUM_COMLETE_EVENT_SIZE
 NET_BUF_POOL_FIXED_DEFINE(sync_evt_pool, 1, SYNC_EVT_SIZE, sizeof(struct bt_buf_data), NULL);
```

Comment:
```suggestion
NET_BUF_POOL_FIXED_DEFINE(sync_evt_pool, 1, BT_BUF_RX_SIZE, sizeof(struct bt_buf_data), NULL);
```

Reply:
not quite, `BT_BUF_RX_SIZE` includes the ACL size too, which might be bigger than 255.

Reply:
Agreed.